lib/librobdb/robdb.spec

```diff
@@ -0,0 +1,64 @@
+Name:           robdb
+Version:        1.1
+Release:        %{autorelease -n %{?dist}}
+Summary:        Provide basic functions to search and read Berkeley Database records
+
+License:        GPL-2.0-or-later
```

Comment:
Below the license is `GPL-2.0-or-later OR LGPL-2.1-or-later`, which is it?

Reply:
Fixed it to GPL-2.0-or-later OR LGPL-2.1-or-later